### DISCUSSION TITLE
Set messages and log window font independently of the entire UI

### DISCUSSION THREAD

**Discussion by jcasale** [texstudio-org/texstudio#3872] [Q&A] [answered]
I'd like to use a console font in the message and log window without setting the default for the entire UI.

Is this possible?

**Comment by octaeder** [comment, marked as answer]
This is not possible from the options dialog. But you could add following last two lines to the .ini file:
```
LogView\WarnIfFileSizeLargerMB=2
LogView\RememberChoiceLargeFile=0
LogView\FontFamily=Arial
LogView\FontSize=9
```
Before changing the .ini file you should end txs. But there is no guaranty that this will work forever.

![grafik](https://github.com/user-attachments/assets/1dd991e8-b0dc-4e72-be42-43ce6cf3bf10)

**Reply by jcasale** [reply]
Thank you very much, is there a full listing that I could use as a reference in the future?

It looks like I can find them in the `*Option()` methods throughout the code at a minimum.

**Reply by octaeder** [reply]
I think there's no list. I had to look into the code. Maybe you need to do so, too.
